newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -221,6 +260,11 @@ const selectLocalResourcePath = (
       paths => {
         if (!paths) return resolve([]);
 
+        // Update stored latestPath value
+        if (paths[0] !== projectPath) {
```

Comment:
Don't do this check, let's always call saveLatestPath - it's the responsibility of this function to do the saving if needed or not. Here we don't care.
We're just saying "Ok we have a new path that was chosen, for this project and this kind of resource. So please save it, I'm giving these information to you - then you do whatever you want with this - not my problem!!" ;)